system-addon/content-src/components/CollapsibleSection/_CollapsibleSection.scss

```diff
@@ -55,8 +55,6 @@
 
     .section-info-option .info-option {
       $arrow-height: 10px;
```

Comment:
This happens to be 10px for all platforms, but let's keep it next to where the other `$os-info-arrow-` variables